### DISCUSSION TITLE
How to use the trigger-volume component in a custom client?

### DISCUSSION THREAD

**Discussion by asathiya007** [mozilla/hubs#3268] [Hubs Cloud Questions] [answered]
Hey everyone! I am trying to create a trigger-box component (a component that can detect collisions - when other objects in the scene enter or leave it) in a custom client. I got my custom client set up through this link: https://hubs.mozilla.com/docs/hubs-cloud-custom-clients.html. After cloning the repo and running it locally, I realized that there is a component called trigger-volume that seems to be what I am looking for. I tried looking around in the existing codebase and there don't seem to be any places where it is being used. Does anyone have any examples or documentation on how this component should be used? I have tried to attach this component to A-Frame entities but I am hitting a dead end and I would appreciate some guidance. Thank you! :)

**Comment by rawnsley** [comment]
I've not used it myself, but I think you can create them if you enable experimental functions in Spoke.

**Reply by asathiya007** [reply]
Okay, I know how to do that in Spoke, but when I run a Hubs custom client, how do I enable experimental features there? [This](https://drive.google.com/file/d/1KSh9k4dcjhYkcGA5v2Lz_U1hrN8udhkC/view?usp=sharing) is what I am seeing right now when I open up my custom client. How can I enable experimental features from this page?

**Comment by pattersonbl2** [comment, marked as answer]
are you still having issues with this?

**Reply by asathiya007** [reply]
No, I was able to figure it out. I just created a new component based on trigger-volume to fit the use case, all good here!
